draft-ietf-quic-transport.md

```diff
@@ -1671,10 +1671,10 @@ Maximum Data:
 
 When counting data toward this limit, an endpoint accounts for the maximum
 offset of data that is sent or received on every stream.  Loss or reordering can
-mean that the maximum offset is greater than the total size of data received on
-a stream.  Receiving STREAM frames might not increase the maximum offset on a
-stream if the maximum offset doesn't increase.  A STREAM frame with a FIN bit
-set or RST_STREAM causes the final offset for a stream to be fixed.
+mean that the maximum offset on a stream can be greater than the total size of
+data received on that stream.  Conversely, receiving STREAM frames might not
+increase the maximum offset on a stream if the maximum offset in the frame is
+lower than the maximum received for that stream.
```

Comment:
I understand now what this says, but it's confusing, since maximum offset and MAX_DATA both represent maxima. I would suggest using "largest received offset" instead of "maximum offset".